docs/user/security/authentication/index.asciidoc

```diff
@@ -383,21 +383,11 @@ xpack.security.authc.providers:
 
 One of the most popular use cases for anonymous access is when you embed {kib} into other applications and don't want to force your users to log in to view it. If you configured {kib} to use anonymous access as the sole authentication mechanism, you don't need to do anything special while embedding {kib}.
 
-If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding, then you will need to add the `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL that you're embedding.
+If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding dashboards or visualizations, then toggle `Public URL` switch presented in the embedding menu.
```

Comment:
What do you think about including a screenshot of this switch within the embedding menu? That would help users discover the feature if they haven't used/seen the embedding menu before.

Reply:
Yep, was hesitating about that (quickly may become outdated and such), but since you mentioned.... I'll add it.

Reply:
Yeah I share your concerns about outdated screenshots. I struggle with the decision every time I write docs, too